parent/pom.xml

```diff
@@ -149,12 +149,14 @@
     <version.parsson>1.1.6</version.parsson>
     <version.springdoc>2.5.0</version.springdoc>
     <version.jakarta.json>2.0.1</version.jakarta.json>
-    <version.node>v20.8.0</version.node>
-    <version.yarn>v1.22.10</version.yarn>
+
+    <version.node>v20.12.2</version.node>
```

Comment:
💭 Would be interesting to know whether this causes any issues with Operate

Reply:
I have already asked the question https://camunda.slack.com/archives/C06HTSPD5AP/p1713510844048369
I have also tested Operate in local, it seems that it should be fine